src/controller/account.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+
+const account = express.Router();
+const { User } = require('../models');
+
+account.get('/user', async (req, res) => {
+  try {
+    const { _id } = req.user;
+    const user = await User.findById(_id, 'username');
```

Comment:
```suggestion
    const user = await User.findById(_id, [ 'username' ]);
```
In case we need more fields in the future